### PR TITLE
Basic Structure

### PR DESCRIPTION
**Pull request by YuvrajM108**
In this pull request I did the following:
- Set up webpack in the project directory and add basic HTML, CSS, and JavaScript files 
- Add HTML elements to index.html file.
- Add basic CSS rules for arranging the elements according to the given wireframe. 
- Set up code linters.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,58 @@
...
+
+To get a local copy up and running follow these simple example steps:
+
+### Install
+
+You can download the code to your personal computer by cloning the repository using the following command:
+
+```
+git clone
+```
```
Comment (carlos-ssh) [inline README.md]: - kindly provide a detailed steps on how to have a copy of your project, including commands.

File: README.md [hunk 2]
```diff
@@ -1 +1,58 @@
...
+### Usage
+
+In order to run this project, enter the following command in your terminal:
+
+```
+npm start
+```
```
Comment (carlos-ssh) [inline README.md:43]: Provide commands like ` npm install ` and `npm run dev`.

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+<body>
+  <h1>Leaderboard</h1>
+  <div class="content">
+    <div class="left-content">
+      <div class="table-header">
+        <h3 class="scores-header">Recent scores</h3>
+        <button class="refresh-btn" id="refresh" type="button">Refresh</button>
+      </div>
+      <table class="scores">
+
```
Comment (carlos-ssh) [inline src/index.html]: - Kindly provide items of your chart: (Name, Score).
leaderboard_wireframe

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @YuvrajM108 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (henatan99) [reply README.md]: This issue is well addressed! :clap:

File: README.md (hunk 2 above)
Comment (henatan99) [reply README.md:43]: This issue is resolved. Well done! :+1:

File: src/index.html (hunk 3 above)
Comment (henatan99) [reply src/index.html]: This issue is well addressed! :dancers:

**Review by henatan99 — APPROVED** [review]
Hi @YuvrajM108 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**